app/dashboard/templates/process_bounty.html

```diff
@@ -103,6 +114,36 @@ <h3>{% trans "Basic Payout" %}</h3>
     document.token_address = '{{bounty.token_address}}';
   </script>
 
+  <style>
+      #tooltip {
```

Comment:
Can we move this to the top or into another css file? cc @thelostone-mc @SaptakS

Reply:
I'd be okay with us moving it to a new file ! let's do that